.github/workflows/lint.yml

```diff
@@ -0,0 +1,33 @@
+name: Lint
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  run-linters:
+    name: Run linters
+    runs-on: ubuntu-20.04
+
+    steps:
+      - name: Check out Git repository
+        uses: actions/checkout@v2.3.4
+
+      - name: Validate composer.json
+        run: composer validate
+
+      - name: Install backend dependencies
+        run: composer install --prefer-dist --no-progress
+
+      - name: Install frontend dependencies
```

Comment:
It seems this installation step ran for 40s. This performance might impact development in the long run, it will make us wait for checks for a good minute each time changes get pushed to the PR branch.

It should be possible to cache installation. I think it's worth it if we want to avoid the wait and spare github servers from processing this request.